proposals/0293-vehicle-type-filter.md

```diff
@@ -388,30 +420,26 @@ The below example shows valid vehicle type filters.
-5. If the proxy determines from `GetVehicleType` ACK that the vehicle type is not supported, the proxy will end the RPC session. The application layer will not be notified about the vehicle type.
-6. If the protocol version does not support the `GetVehicleType` protocol message, the SDL proxy will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
-7. If the `GetVehicleType` message response is NACK, the proxy will end the RPC session and the app will not be registered on the SDL system.
-
-![Sequence Diagram](../assets/proposals/0293-vehicle-type-filter/ios.png)
+1. The proxy will receive vehicle type info in `StartServiceAck` protocol message. 
+2. On receiving vehicle type information, proxy will check `SDLSupportedVehicleTypes` keys to check supported vehicle types.
+3. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. 
+4. If the proxy determines that connected vehicle type is not supported, the proxy will end the RPC session. The application will not register on SDL enabled IVI system.
+5. If the vehicle type information is not available in `StartServiceAck` protocol message, the SDL proxy will continue with the app registration and it will rely on vehicle type information received in the `RegisterAppInterface` response. If vehicle type is not supported, the exclusive apps will be allowed to unregister from the SDL enabled system.
 
 ## Potential downsides
 
-The Android Router Service after transport connection needs to start the RPC service. If the Android Router Service does not start RPC service and relies on the client to send the `GetVehicleType` protocol message, the connected client would start another foreground service. This would force the connected app to show two notifications to users while determining vehicle type information and would need to terminate services if the app does not support the SDL enabled IVI system. 
+The SDL device Listener after transport connection needs to start the RPC service. If the SDL device Listener does not start RPC service and relies on the client for vehicle type details, the connected client would start another foreground service. This would force the connected app to show two notifications to users while determining vehicle type information and would need to terminate services if the app does not support the SDL enabled IVI system. 
```

Comment:
```suggestion
The SDL Device Listener after transport connection needs to start the RPC service. If the SDL Device Listener does not start the RPC service and relies on the client for vehicle type details, the connected client would start another foreground service. This would force the connected app to show two notifications to users while determining vehicle type information and would need to terminate services if the app does not support the SDL enabled IVI system. 
```